src/connectors/breadcrumb/__tests__/connectBreadcrumb-testing-library-test.js

```diff
@@ -0,0 +1,245 @@
+import { wait } from '@testing-library/dom';
+
+import instantsearch from '../../../index';
+
+import apiResults from '../../../widgets/breadcrumb/__tests__/__fixtures__/results.json';
+
+const searchClient = {
+  search: jest.fn(() => Promise.resolve(apiResults)),
+};
+
+describe('connectBreadcrumb', () => {
+  let search;
+
+  beforeEach(() => {
+    search = instantsearch({
+      indexName: 'instant_search',
+      searchClient,
+    });
+  });
+
+  afterEach(() => {
+    try {
+      search.dispose();
+    } catch (e) {
+      // Fail if the search was never started, ignoring
```

Comment:
can wrap in if (search.started) instead

Reply:
We don't want to use `search.started` because it's an internal.

Reply:
It seems publicly exposed, but yes it is not documented :neutral_face: 

Also while making the change to use `search.started` I found an issue in `connectBreadcrumb`. If you do not provide an `unmountFn` during widget creation then `search.dispose()` crashes (no default value).

Reply:
@yannickcr Some connectors still don't provide a default value for the unmount function. This is known but we haven't fixed that yet.

Regarding `search.started`, I don't think we should use it because it is not supposed to be exposed. This is an internal variable accessible because we use a class for `InstantSearch` and we store `started` on `this`. If we were to refactor this class to a function or even rename `started`, this will fail. I believe we should _strictly_ restrict ourselves to the public API with this move.

Reply:
Agree to restrict ourselves to the public API